ragstack-e2e-tests/e2e_tests/langchain/test_compatibility_rag.py

```diff
@@ -59,17 +60,32 @@ def _chat_openai(**kwargs) -> ChatOpenAI:
 
 @pytest.fixture
 def openai_gpt35turbo_llm():
-    return _chat_openai(model="gpt-3.5-turbo", streaming=False)
+    # NeMo guardrails fails for this model with the given prompts.
```

Comment:
why it fails ?

Reply:
The prompts used by guardrails are sometimes misunderstood by the model. Syncing with their team to see how to properly use the rails and give them the best chance for success.